openedx/core/djangoapps/bookmarks/tests/test_tasks.py

```diff
@@ -152,6 +154,12 @@ def test_update_xblocks_cache(self, course_attr, expected_sql_queries):
         """
         course = getattr(self, course_attr)
 
+        if settings.ROOT_URLCONF == 'lms.urls':
+            # When the tests run under LMS, there is a certificates course_published
+            # signal handler that runs and causes the number of queries to be one more
+            # (due to the check for disabled_xblocks in django.py).
+            expected_sql_queries += 1
```

Comment:
Do you know why this test is running under different envs? I realize that fixing that may be beyond the scope of your PR, so I don't consider this a blocker. But it's unfortunate. :-(